pkg/metrics/metrics.go

```diff
@@ -11,11 +12,26 @@ var (
 	registeredCollectors  = []prometheus.Collector{}
 	initializedCollectors = make(map[string]struct{})
 	log                   = logger.NewLogger("collectors")
+	policyMetrics         string
```

Comment:
nitpick: regarding my earlier comment about avoiding double-conversion, this could be stored as `[]byte` instead.